duckduckgo/duckduckgo-privacy-extension
# new svg grade icons

**Reviewer:**



## Description:

The current grade icons don't look great with dark browser themes. I added colored svg icons. 

Before
<img width="61" alt="Screen Shot 2019-03-15 at 11 38 19 AM" src="https://user-images.githubusercontent.com/1882892/54450775-2c202680-4749-11e9-9e84-030c51ca2994.png">

After
<img width="35" alt="Screen Shot 2019-03-15 at 11 38 56 AM" src="https://user-images.githubusercontent.com/1882892/54451211-368ef000-474a-11e9-95ed-25e93d078718.png">

## Steps to test this PR:
1. Build and install extension. 
2. Test on some sites. You should have colored grade icons. 
3. Repeat on both light and dark themes

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
